src/mlpack/core/math/quantile.hpp

```diff
@@ -0,0 +1,123 @@
+/**
+ * @file core/math/quantile.hpp
+ * @author Shubham Agrawal
+ *
+ * Miscellaneous math quantile-related routines.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_MATH_QUANTILE_HPP
+#define MLPACK_CORE_MATH_QUANTILE_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace math /** Miscellaneous math routines. */ {
+
+/**
+ * Computes the inverse erf function using the rational approximation from
+ * Numerical Recipes.
+ * Code accompanying the article "Approximating the erfinv function" in 
+ * GPU Computing Gems, Volume 2
+ *
+ * @param x Input value.
+ */
+inline double erfinverse(double x)
```

Comment:
I compared this to https://github.com/lakshayg/erfinv/blob/master/erfinv.c and the implementation comes with an extra refine step; so it might be a good idea to compare this function against the boost variant, to check how close it matches the implementation.

Reply:
I think here refine will not work, and refine will be time-consuming.

Reply:
Can you do a comparison between your implementation and the boost implementation?